### PR TITLE
Williamhe/1853186 comment request

### PR DESCRIPTION
**Pull request by william-msft**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as:
  - Resolving [this comment](https://github.com/microsoft/Oryx/pull/2072#discussion_r1269736580)
- [x] Tests are included and/or updated for code changes.
  - [dotnet ](https://github.com/microsoft/Oryx/actions/runs/5663038303/job/15344074089#step:4:1)sdk automation test
  - [python ](https://github.com/microsoft/Oryx/actions/runs/5663063585/job/15344148706#step:4:1)sdk automation test

~- [ ] Proper license headers are included in each file.~

### CODE REVIEW

File: build/tools/Automation/Commons/SdkStorageHelper.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+﻿using System;
```
Comment (cormacpayne) [inline build/tools/Automation/Commons/SdkStorageHelper.cs:1]: nit: let's add a license header to this file

File: build/tools/Automation/Commons/SdkStorageHelper.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            if (!string.IsNullOrEmpty(blockedVersions))
+            {
+                var versionStrings = blockedVersions.Split(',');
+                foreach (var versionString in versionStrings)
+                {
+                    result.Add(versionString.Trim());
+                }
+            }
+
+            return result;
```
Comment (cormacpayne) [inline build/tools/Automation/Commons/SdkStorageHelper.cs:59]: nit: we can condense this entire method a bit using LINQ:

```cs
return string.IsNullOrEmpty(blockedVersions) ?
    new List<string>() :
    blockedVersions.Split(',').Select(v => v.Trim()).ToList();
```

**Review by cormacpayne — DISMISSED** [review]
Two minor comments, other LGTM :shipit:

File: build/tools/Automation/Commons/SdkStorageHelper.cs (hunk 1 above)
Comment (william-msft) [reply build/tools/Automation/Commons/SdkStorageHelper.cs]: Updated.

File: build/tools/Automation/Commons/SdkStorageHelper.cs (hunk 2 above)
Comment (william-msft) [reply build/tools/Automation/Commons/SdkStorageHelper.cs]: Updated

**Review by cormacpayne — DISMISSED** [review]

**Comment by william-msft** [comment]
Triggered new tests:
- dotnet [test](https://github.com/microsoft/Oryx/actions/runs/5765383777/job/15631226623#step:4:1)
- python [test](https://github.com/microsoft/Oryx/actions/runs/5663063585/job/15344148706#step:4:1)

**Review by cormacpayne — APPROVED** [review]

**Pull request merged**